cms/templates/maintenance/index.html

```diff
@@ -0,0 +1,17 @@
+<%page expression_filter="h"/>
+<%inherit file="base.html" />
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+%>
+<%block name="title">${_('Maintenance Dashboard')}</%block>
+<%block name="viewcontent">
+<section class="container maintenance-content">
```

Comment:
If this is truly a ```<section>``` then it should have a heading as its first child that describes the theme of the content.  Perhaps you should make the two blocks that precede this children of the section element.  That would work.